### ISSUE TITLE
Fix install targets after destdir introduction

### ISSUE THREAD

**Issue by ymartin59** [SynoCommunity/spksrc#3753]
Review corresponding package:
```
./cross/pocketsphinx/Makefile:INSTALL_TARGET = install_target install_python_wheel
./cross/sphinxbase/Makefile:INSTALL_TARGET = install_target install_python_wheel
./cross/umurmur/Makefile:INSTALL_TARGET = install_target
```

Where `install_target` should be replaced by `install_destdir_target`, the same way as commit for `deluge`: https://github.com/SynoCommunity/spksrc/commit/536c78398ed1831afab9163713d110bc9bcd9eb1

**Comment by hgy59** [comment]
@ymartin59 for uMurmur it must be `INSTALL_TARGET = install_target` or we define
```
umurmur_install:
    $(RUN) $(MAKE) install prefix=$(INSTALL_PREFIX)
```
with the default `$(RUN) $(MAKE) DESTDIR=$(INSTALL_DIR) install prefix=$(INSTALL_PREFIX)`

the install folder would be
`/spksrc/spk/umurmur/work-x64-6.1/install/spksrc/spk/umurmur/work-x64-6.1/install/var/packages/umurmur/target/bin`

instead of
`/spksrc/spk/umurmur/work-x64-6.1/install/var/packages/umurmur/target/bin`.

i.e. it is two time prefixed with `$(WORK_DIR)/install`

May be it would be better to define a custom `INSTALL_TARGET` in the spk/umurmur/Makefile instead of using the no so obvious `install_target` defined in spksrc.install.mk

**Comment by hgy59** [comment]
This is fixed some time ago (#3998) by migration from cc to cmake build system.
